plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -54,6 +54,7 @@ class DbtConfigArgs:
     project_dir: Optional[str] = None
     profiles_dir: Optional[str] = None
     profile: Optional[str] = None
+    single_threaded: bool = False
```

Comment:
What's this for?

Reply:
@barrywhart let's try removing this in my PR and see what happens 😛

Reply:
![image](https://user-images.githubusercontent.com/80432516/148617413-0bf095b3-5566-49e3-87bd-30c5533fc66c.png)
assume it is an option dbt takes, but it's false by default so can probs just remove

Reply:
Ooh okay it threw an error when removed let's leave it 🤣 
![image](https://user-images.githubusercontent.com/80432516/148617728-a0254877-3db5-4ef0-8131-91debf17fdb7.png)